source/fundamentals/databases-collections.txt

```diff
@@ -0,0 +1,173 @@
+
+For more information, see the :manual:`server documentation for capped
+collections </core/capped-collections>`.
+
+Document Validation
+~~~~~~~~~~~~~~~~~~~
+
+**Document validation** provides the ability to validate documents
+against a series of filters during writes to a collection. You can
+specify these filters using the :java-docs:`ValidationOptions
+<apidocs/mongodb-driver-core/com/mongodb/client/model/ValidationOptions.html>`
+class, which accepts a series of :java-docs:`Filters
+<apidocs/mongodb-driver-core/com/mongodb/client/model/Filters.html>`
+that specifies the validation rules and expressions:
+
+.. code-block:: java
+
+   ValidationOptions collOptions = new ValidationOptions().validator(
+        Filters.or(Filters.exists("commander"), Filters.exists("first officer")));
+   database.createCollection("ships",
+        new CreateCollectionOptions().validationOptions(collOptions));
+
+For more information, see the :manual:`server documentation for document
+validation </core/document-validation>`.
+
+Get a List of Collections
+-------------------------
+
+You can query for a list of collections in a database using the
+:java-sync-api:`MongoDatabase.listCollectionNames()
+<com/mongodb/client/MongoDatabase.html#listCollectionNames()>` method:
+
+.. code-block:: java
+
+   for (String name : database.listCollectionNames()) {
+      System.out.println(name);
+   }
+
+Drop a Collection
+-----------------
+
+You can remove a collection from the database using the 
+:java-sync-api:`MongoCollection.drop()
+<com/mongodb/client/MongoCollection.html#drop()>` method:
+
+.. code-block:: java
+
+   MongoCollection<Document> collection = database.getCollection("bass");
+   collection.drop();
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database also implicitly removes all
+   documents that previously belonged to that collection. Only drop
+   collections that contain data that is no longer needed.
+
+Specify Read Preferences, Read Concerns and  Write Concerns
+-----------------------------------------------------------
+
+**Read preferences**, **read concerns**, and **write concerns** control
+how the driver routes read operations and waits for acknowledgement for
+read and write operations when connected to a MongoDB replica set.
+
+Read preferences, read concerns, and write concerns apply to all
+read and write operations, and default settings are shared across entire
+MongoDB instance connections at the MongoClient level.
+
+However, if your application requires a different read preference, read
+concern, or write concern for a ``MongoDatabase`` or ``MongoCollection``,
+you can use the following methods to obtain an instance of a collection
+or database with your preferred preference or concern:
+
+- :java-sync-api:`MongoDatabase.withReadConcern() <com/mongodb/client/MongoDatabase.html#withReadConcern(com.mongodb.ReadConcern)>`
```

Comment:
No need to update this, but I bet a table containing this info would look pretty organized without taking up too much vertical space. There might even be room, horizontally, for a column for the server manual links.

Reply:
This is tempting, but I'm not sure if it makes much sense to separate the links from the methods themselves. I'll change this if the next staging build looks messy to me.

Reply:
Not sure I understand as the table would consolidate them. It would contain columns for:
- Option name,  Description, Client scope method, Database scope method, Collection scope method

Just an idea, no need for action